src/salt_test/__init__.py

```diff
@@ -30,7 +31,7 @@
 }
 
 FLAVOR_RPM = {
-    "classic": "python3-salt-testsuite",
+    "classic": "salt-testsuite",
```

Comment:
I don't like this change. This dict is now coupled to `_get_classic_pkg_name`, it can't be used on its own.

```suggestion
    "python3": "python3-salt-testsuite",
    "python311": python311-salt-testuite",
```
would be closed to extension (requires code changes), but keeps the rest of this wrapper simple.